Classes/Utility/GeneralUtility.php

```diff
@@ -928,27 +952,20 @@ public static function parseConditionsBlock(array $settings, array $gp): array {
     }
     foreach ($settings['if.'] as $idx => $conditionSettings) {
       $conditionSettings = is_array($conditionSettings) ? (array) $conditionSettings : [];
-      $conditions = $conditionSettings['conditions.'] ?? [];
-      $orConditions = [];
-      foreach ($conditions as $subIdx => $andConditions) {
-        $results = [];
-        foreach ($andConditions as $subSubIdx => $andCondition) {
-          $result = strval(self::getConditionResult($andCondition, $gp));
-          $results[] = ($result ? 'TRUE' : 'FALSE');
-        }
-        $orConditions[] = '('.implode(' && ', $results).')';
-      }
-      $finalCondition = '('.implode(' || ', $orConditions).')';
 
-      $evaluation = false;
-      eval('$evaluation = '.$finalCondition.';');
+      $evaluation = self::evaluateConditions($conditionSettings, $gp);
 
       // @phpstan-ignore-next-line
```

Comment:
Now that `$evaluation` has type bool and is not always false for phpstan, remove ignore as it is not needed anymore.